options/linux/include/sys/mount.h

```diff
@@ -40,6 +40,11 @@ extern "C" {
 
 #ifndef __MLIBC_ABI_ONLY
 
+#define MNT_FORCE 1
+#define MNT_DETACH (1<<1)
+#define MNT_EXPIRE (1<<2)
```

Comment:
We should move these defines outside the MLIBC_ABI_ONLY define. Also please make sure these new defines match Linux. Better yet, we should really move these to an abi-bit header, as I'm pretty sure that at least some of the constants in this file don't match Linux.